HerPublicWebsite.BusinessLogic/Services/CsvFileCreator/CsvFileCreator.cs

```diff
@@ -268,8 +268,7 @@ public CsvRowReferralRequest(ReferralRequest request)
             };
             HouseholdIncome = request.IncomeBand switch
             {
-                //Obsolete Income Bands used to preserve backwards-compatibility
-                #pragma warning disable CS0618
+                #pragma warning disable CS0618 // Obsolete Income Bands used to preserve backwards-compatibility
```

Comment:
This one wants to be un-tabbed as well (unless it's just a visual thing on Gitlab's end)

Reply:
Ah, missed one ^^'